dart/lib/src/sentry_span_context.dart

```diff
@@ -0,0 +1,80 @@
+// import 'package:meta/meta.dart';
+
+import '../sentry.dart';
+
+// @immutable
+// SpanContext is the attribute collection for a Span (Can be an implementation detail). When possible SpanContext should be immutable.
+// status and sampled should be mutable, so how we do it? clone it?
```

Comment:
to discuss

Reply:
to check: the mutable fields should not belong to the SpanContext then but rather to the SentrySpan itself